jetstream/export_json.py

```diff
@@ -35,9 +35,11 @@ def _get_statistics_tables_last_modified(
     return {row.table_id: row.last_modified for row in result}
 
 
-def _get_gcs_blobs(storage_client: storage.Client, bucket: str) -> Dict[str, datetime]:
+def _get_gcs_blobs(
+    storage_client: storage.Client, bucket: str, target_path: str
+) -> Dict[str, datetime]:
     """Return all blobs in the GCS location with their last modified timestamp."""
-    blobs = storage_client.list_blobs(bucket)
+    blobs = storage_client.list_blobs(bucket, prefix=target_path + "/")
```

Comment:
Why the `"/"`? `prefix` checks if the path following after the bucket name matches the provided string. So it would also match `gs://mozanalysis/statistics_some_cool_experiment_weekly` and not just the ones in the `statistics/` subdirectory.